meta/tastaturberuf/contao-image-copyright-bundle/es.yml

```diff
@@ -0,0 +1,20 @@
+es:
+    title: Información sobre la licencia de las imágenes
+    description: >
+        Con este paquete de Contao, puede añadir fácilmente un aviso de licencia para cada imagen y luego mostrarlas en un módulo.
+        Especialmente útil si utiliza fotos de stock de varios proveedores y necesita mostrar la información de la licencia.
```

Comment:
`stock` needs to be added to the `es` allow list.

Reply:
All checks have passed